cmake/onnxruntime_unittests.cmake

```diff
@@ -447,7 +447,7 @@ if(HAS_FILESYSTEM_H OR HAS_EXPERIMENTAL_FILESYSTEM_H)
     target_compile_options(onnxruntime_perf_test PRIVATE ${disabled_warnings})
   endif()
   onnxruntime_add_include_to_target(onnxruntime_perf_test gsl)
-  target_link_libraries(onnxruntime_perf_test PRIVATE ${GETOPT_LIB} ${onnx_test_libs})
+  target_link_libraries(onnxruntime_perf_test PRIVATE ${GETOPT_LIB} ${onnx_test_libs} ${trt_link_libs})
```

Comment:
same comment.

Reply:
I've moved it to onnxruntime_unittests_internal.cmake